content/2019/personal-notetaking-in-vim/index.md

```diff
@@ -0,0 +1,343 @@
+  freenode: "swalladge"
+  name: "Samuel Walladge"
+  picture: "https://www.swalladge.net/assets/images/logo.jpg"
+  twitter: "@srwalladge"
+  dotfiles: https://github.com/swalladge/dotfiles
+---
+
+> It pays you not to blink sometimes. It gives you a heck of a fright.
+>
+> – My grandmother, on rapid change
+
+
+## Intro
+
+Part of Vim's power is how it can integrate with its environment. It can
+interact with external programs, external scripts can interact with Vim, and
+Vim is of course scriptable. Here, I'm going to detail an example of weaving
+Vim with other applications and the environment to implement a notetaking
+methodology that I personally use.
+
+My notetaking has changed a lot over the years, from little to no notetaking (I
+don't recommend!), to experimenting with off-the-shelf tools
+([Simplenote][simplenote], [Standardnotes][standardnote], [Vimwiki][vimwiki]),
+to experimenting with various knowledge management methodologies, to rolling my
+own framework for [Zettelkasten][zettelkasten] in Vim. I haven't yet reached
+the ideal setup, but it's feeling close, and it's reached a point where I can
+comfortably.
+
+
+### Sidenote: Create + Consume in Vim
+
+Yes, you did see non-Vim tools in my list of previously used tools for notes...
+It should be noted that I was using Vim as my main editor long before I
+switched to using Vim to take notes. The reason why is that for a time I knew I
+wanted to create and edit notes using Vim, but I wanted to be able to read
+those notes in other ways, such as rich text from rendered markdown
+(Simplenote), or a wiki that was navigable in a browser (Vimwiki). The times I
+used Vim to edit was frustrating, because I needed to fit with a workflow that
+required rebuilding a wiki after editing, or opening one file at a time for
+something like Simplenote.
+
+Finally, I realized that I didn't actually need to read rich text or navigate
+hyperlinks with a mouse in a browser. I could create *and consume* in Vim!
+Granted, it's not as pretty for viewing sometimes, but now creating, editing,
+searching, and reading notes are all the same thing. Efficiency.
+
+
+### Zettelkasten
+
+One more thing to discuss before we dive in - what is Zettelkasten?.
+Zettelkasten (German for "card index") is a method for personal knowledge
+management in which one uses many small atomic notes, linked to other such
+notes. The idea is that it forms a huge interconnected network of notes one can
+traverse and interact with. I came across this methodology a little while back
+and love the idea. I haven't yet spent the time to really learn how to use it
+effectively, but I've started to use some of the ideas, including linking
+between notes and facilitating easy creation of notes.
+
+If you're interested to know more about Zettelkasten as a system, see
+[zettelkasten.de][zettelkasten] and/or web search for it. There are many good
+resources. What I described above is a horrendous simplification.
```

Comment:
`an horrendous`, but really it's not horrendous, it's just a sensible simplification, so maybe just remove `horrendous`.